ee/vellum_ee/workflows/display/workflows/get_vellum_workflow_display_class.py

```diff
@@ -11,6 +12,19 @@
 
 
 def _get_workflow_display_class(*, workflow_class: Type[WorkflowType]) -> Type["BaseWorkflowDisplay"]:
+    # Try to import the display module to ensure display classes are registered
+    try:
+        # Extract package root from workflow module (e.g., "foo.workflow" -> "foo")
+        workflow_module = workflow_class.__module__
+        if workflow_module.endswith(".workflow"):
+            package_root = workflow_module[: -len(".workflow")]
+        else:
+            package_root = workflow_module
+        display_module_path = f"{package_root}.display"
```

Comment:
This is interesting. Actually I think we could just do this in the loader itself